selectrum.el

```diff
@@ -356,6 +363,24 @@ If PREDICATE is non-nil, then it filters the collection as in
   (or (get-text-property 0 'selectrum-candidate-full candidate)
       candidate))
 
+(defvar selectrum--minibuffer-default-prompt-regexps
+  (let ((minibuffer-eldef-shorten-default nil))
+    (cl-remove-if (lambda (i) (and (consp i) (nth 2 i)))
+                  (minibuffer-default--in-prompt-regexps)))
+  "See `minibuffer-default-in-prompt-regexps'.")
```

Comment:
Handy variable, thanks for making me aware of it! (I mean obviously the existence of that whole library is a giant hack, but we might as well use it since it's not going anywhere.)

Reply:
Yes, but this is Emacs right? One gigantic hack :laughing: